maapcopcguide (2).ipynb

```diff
@@ -0,0 +1,290 @@
+{
```

Comment:
<p>Change this to something like - Working with LiDAR Cloud Optimized Point Cloud (COPC) in MAAP</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/459/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>